pkg/autoscaler/scaling/autoscaler_test.go

```diff
@@ -65,6 +66,44 @@ func expectedEBC(totCap, targetBC, recordedConcurrency, numPods float64) int32 {
 	return int32(math.Floor(totCap/targetUtilization*numPods - targetBC - recordedConcurrency))
 }
 
+func expectedExcessBCF(originalReadyPodsCount int, totalValue, observedStableValue, targetBurstCapacity float64) float64 {
+	excessBCF := math.Floor(float64(originalReadyPodsCount)*totalValue - observedStableValue -
+		targetBurstCapacity)
+	return excessBCF
+}
+
+func TestAutoscalerMetrics(t *testing.T) {
+	metricClient := &autoscalerfake.MetricClient{StableConcurrency: 50.0, PanicConcurrency: 50.0, StableRPS: 100}
+	a := newTestAutoscaler(t, 10, 100, metricClient)
+	a.expectScale(t, time.Now(), 5, expectedEBC(10, 100, 50, 1), true)
+	spec, podCounter := a.currentSpecAndPC()
+	originalReadyPodsCount, _ := podCounter.ReadyCount()
+	wantTags := map[string]string{}
```

Comment:
Shouldn't the tags be non-empty?
I think we create the autoscaler with empty context, but for this test we probably want a one with tags set to make sure we use the correct one.

Reply:
Done